wicket-bean-validation/src/main/java/org/apache/wicket/bean/validation/PropertyValidator_fr.properties.xml

```diff
@@ -26,6 +26,8 @@
 	<entry key="javax.validation.constraints.Max.message">Le champ '${label}' doit être inférieur ou égal à ${value}</entry>
 	<entry key="javax.validation.constraints.Min.message">Le champ '${label}' doit être supérieur ou égal à ${value}</entry>
 	<entry key="javax.validation.constraints.NotNull.message">Le champ '${label}' ne doit pas être null</entry>
+	<entry key="javax.validation.constraints.NotBlank.message">Le champ '${label}' ne peut pas être vide</entry>
```

Comment:
A couple of other annotations have been added to [BV 2.0](http://beanvalidation.org/2.0/) as well:

> new built-in constraints: @Email, @NotEmpty, @NotBlank, @Positive, @PositiveOrZero, @Negative, @NegativeOrZero, @PastOrPresent and @FutureOrPresent

@solomax: Should we add translations for these new constraints?

Reply:
+1 for new labels, I'll add `_ru` localization ...

Reply:
Added properties for English and French. My French is a bit rusty, but the translations should be ok.

Reply:
We should probably copy the properties from the HV implementation:

https://github.com/hibernate/hibernate-validator/tree/master/engine/src/main/resources/org/hibernate/validator

Reply:
It is APL licensed so, I guess, we can copy
I would better use existing ones, but not sure how is this possible :(

Reply:
The existing ones could probably be loaded via a custom `IStringResourceLoader` but this is out of the scope of this PR.

@solomax: What else do you want me to do for this PR so it can be merged?

Reply:
OK with me :)